src/main/scala/io/github/paoloboni/binance/fapi/FutureApi.scala

```diff
@@ -111,6 +112,74 @@ final case class FutureApi[F[_]: Async: WithClock: LogWriter](
     } yield prices
   }
 
+  def getPrice(getPriceParams: PriceTickerParams): F[Price] = {
+    val url = Url(
+      scheme = config.scheme,
+      host = config.host,
+      port = config.port,
+      path = "/fapi/v1/ticker/price"
+    )
+
+    for {
+      price <- client.get[Price](
+        url = url,
+        limiters = rateLimiters.filterNot(_.limitType == common.response.RateLimitType.ORDERS)
+      )
+    } yield price
```

Comment:
is the for-comprehension unnecessary here?
```suggestion
    client.get[Price](
         url = url,
         limiters = rateLimiters.filterNot(_.limitType == common.response.RateLimitType.ORDERS)
       )
```

Reply:
yes, it absolutely is unnecessary